plugins/LanguagesManager/Model.php

```diff
@@ -57,10 +57,41 @@ public function setLanguageForUser($login, $languageCode)
         return true;
     }
 
+    /**
+     * Returns whether the given user has choosen to use 12 hour clock
+     *
+     * @param $userLogin
+     * @return bool
+     * @throws \Exception
+     */
+    public function uses12HourClock($userLogin)
```

Comment:
I reckon it would be nice to have a simple test for this method and the following but maybe not needed